### ISSUE TITLE
Support arrays fields

### ISSUE THREAD

**Issue by sporniket** [sporniket/javabeans#35]
**When** there is the following declaration : 

```java
String[] refFields;
```

**Then** there is the following generated code : 

```java
    private String[] myRefFields ;

    public String[] getRefFields() {return myRefFields ;}
    public void setRefFields(String[] value) {myRefFields = value;}

```

Instead of the following generated code :

```java
    private String myRefFields ;

    public String getRefFields() {return myRefFields ;}
    public void setRefFields(String value) {myRefFields = value;}
```

**Comment by sporniket** [comment]
Technically it's a missing feature, but as it was a bad surprise when trying to generate as Javabean with such field, it is also a bug from the user perspective.

**Comment by sporniket** [comment]
Misreferenced, fixed by bde75dd7da2e590850f362fd63a20883ceae08a2
